test-projects/basic/server.js

```diff
@@ -7,7 +7,7 @@ const initialData = require('./data');
 keystone
   .prepare({ apps, port, dev: process.env.NODE_ENV !== 'production' })
   .then(async ({ middlewares }) => {
-    await keystone.connect(process.env.MONGODB_URI);
+    await keystone.connect();
```

Comment:
Turns out this wasn't actually being set anywhere.

Reply:
@jesstelford it is informative to have as part of demo though.

Reply:
It's since been replaced by `CONNECT_TO` in #1201